exchange/bidder_test.go

```diff
@@ -1401,6 +1422,23 @@ func TestTimeoutNotificationOn(t *testing.T) {
 	assert.EqualValues(t, logExpected, logActual)
 }
 
+func TestParseDebugInfoTrue(t *testing.T) {
+	debugInfo := &config.DebugInfo{Allow: true}
+	resDebugInfo := parseDebugInfo(debugInfo)
+	assert.True(t, resDebugInfo.Allow, "Debug Allow value should be false")
```

Comment:
The failure message here should mention true instead of false

Reply:
Fixed, thank you!